src/sources/splunk_hec/mod.rs

```diff
@@ -226,7 +227,8 @@ impl SourceConfig for SplunkConfig {
             LogNamespace::Vector => vector_lib::schema::Definition::new_with_default_metadata(
                 Kind::bytes().or_object(Collection::empty()),
                 [log_namespace],
-            ),
+            )
+            .with_meaning(OwnedTargetPath::event_root(), meaning::MESSAGE),
```

Comment:
Did you mean to put this here? I think it should only apply to `LogNamespace::Vector`.  `LogNamespace::Legacy` already sets it.

Reply:
Nvm, I got confused by the indentation.